src/components/Map.js

```diff
@@ -20,8 +20,8 @@ export const MapContext = React.createContext(null);
 function Map() {
   const mapContainer = useRef(null);
   const map = useRef(null);
-  const [lng, setLng] = useState(-2.25);
-  const [lat, setLat] = useState(53.48);
+  const [lng, setLng] = useState(-2.6);
+  const [lat, setLat] = useState(51.45);
```

Comment:
Centering the map on Bristol - hope that's okay!

Reply:
Makes sense! The previous coords were Manchester; also arbitrary.